packages/sftp-client/tests/test-index.js

```diff
@@ -38,14 +40,18 @@ test('SftpClient supports password authentication', async (t) => {
     password: 'password'
   });
 
+  let files;
   try {
-    const files = await sftpClient.list('/');
-    const fileNames = files.map((f) => f.name);
+    await sftpClient.connect();
```

Comment:
Same question: why not use the client from `t.context`?

Reply:
It is testing a different auth scheme, which requires a new client.